etc/docker/Dockerfile.ubuntu.focal

```diff
@@ -49,7 +49,7 @@ RUN apt-get -y install cmake
 RUN mkdir -p ${HOME}/opt/src
 
 RUN cd ${HOME}/opt/src && \
-    git clone --recurse-submodules -b v1.52.0 --depth 1 --shallow-submodules https://github.com/grpc/grpc && \
+    git clone --recurse-submodules -b v1.62.1 --depth 1 --shallow-submodules https://github.com/grpc/grpc && \
```

Comment:
This is the only docker image where we build grpc/protobuf from source so it makes sense to keep up with the latest stable on it, since we can.